Formula/prettyping.rb

```diff
@@ -6,6 +6,9 @@ class Prettyping < Formula
 
   bottle :unneeded
 
+  # Fix IPv6 handling on BSD/OSX: https://github.com/denilsonsa/prettyping/pull/11
+  patch :DATA
```

Comment:
Afraid not, I tried that. There's been a few changes to master since the 1.0.0 release so the hunks all fail if you try that.